chrome/otp.js

```diff
@@ -0,0 +1,30 @@
+var totpInput = document.getElementById("totp");
```

Comment:
Let's use `otp` instead of `totp` everywhere, as it was suggested that it could be `hotp` as well.

Reply:
Looks like I forgot about that. Will fix.